src/static_canvas.class.js

```diff
@@ -1177,7 +1177,7 @@
         objects: this._toObjects(methodName, propertiesToInclude),
       };
       if (clipPath) {
-        data.clipPath = this._toObjectMethod(clipPath, methodName, propertiesToInclude);
+        data.clipPath = this.clipPath;
```

Comment:
wait, this should call _toOject anyway no?
In this way we are exporting the instance itself.
We want it's object representation.